tools/pipelines/build-api-markdown-documenter.yml

```diff
@@ -32,6 +32,7 @@ trigger:
   paths:
     include:
     - .prettierignore
+    - common/build/eslint-config-fluid # Required since api-markdown-documenter takes a `file:` dependency on this package
```

Comment:
This is really unrelated to the core changes being made here, but it fixes a gap in our pipeline trigger conditions.